docs/building.md

```diff
@@ -12,7 +12,7 @@ This part assumes you are not interested in building the Android or iOS wrappers
 The following command will ensure your environment variables are set properly and build the dependencies needed by the project:
 
 ```
-./libs/verify-desktop-environment.sh
+cargo verify_desktop
```

Comment:
if someone reads `building.md` before reading `docs/contributing.md` they might not have have `asdev` installed and get 
```
error: no such subcommand: `asdev`

	Did you mean `audit`?
```

Maybe worth repeating what you have in contributing.md  in a shorter form?